pymc/variational/opvi.py

```diff
@@ -1127,28 +938,6 @@ def _new_initial(self, size, deterministic, more_replacements=None):
             initial = at.switch(deterministic, at.ones(shape, dtype) * dist_map, sample)
             return initial
 
-    @node_property
-    def symbolic_random(self):
```

Comment:
this was an important abstract method, should not be deleted

Reply:
but it raise only `NotImplementedError` is that right ?